riff-raff.yaml

```
@@ -0,0 +1,17 @@
+stacks: [media-service]
```

Definitely overkill but a good excuse to use this: https://docs.github.com/en/actions/security-guides/encrypted-secrets
```suggestion
stacks: ${{ secrets.STACK}}
regions: ${{ secrets.AWS_REGION }}
```

---

Better docs: https://github.com/Azure/actions-workflow-samples/blob/master/assets/create-secrets-for-GitHub-workflows.md

---

The `riff-raff.yaml` file is not a Github Actions yaml file, so it will never resolve those secrets. Good instincts though to keep security front of mind